yara-x/src/modules/time.rs

```diff
@@ -0,0 +1,41 @@
+use crate::modules::prelude::*;
+use crate::modules::protos::time::*;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[module_main]
+fn main(_ctx: &ScanContext) -> Time {
+    // Nothing to do, but we have to return our protobuf
+    let time_proto = Time::new();
+    time_proto
+}
+
+#[module_export(name = "now")]
+fn time_now(ctx: &ScanContext) -> Option<i64> {
```

Comment:
You don't need to use `name ="now"`, naming the function `now` is enough, like this...

```rust
#[module_export]
fn now(ctx: &ScanContext) -> Option<i64> { ... }
```

Reply:
Thanks, will change to:

```
#[module_export]
fn now(ctx: &ScanContext) -> Option<i64> {
```

Reply:
Done